app/src/main/java/pl/targosz/invoicing/services/JsonService.java

```diff
@@ -0,0 +1,30 @@
+package pl.targosz.invoicing.services;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+public class JsonService<T> {
+
+    private final ObjectMapper objectMapper;
+
+    public JsonService() {
+        objectMapper = new ObjectMapper();
+        objectMapper.findAndRegisterModules();
+        objectMapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+    }
+
+    public String toJson(final T object) throws JsonProcessingException {
+        try {
+            return objectMapper.writeValueAsString(object);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+            return null;
```

Comment:
Returning null in this kind of methods is not a good practice - its better to throw exception

Reply:
Done